### PR TITLE
Dmitri/new types

### PR DESCRIPTION
**Pull request by a-palchikov**
This changes adds support for new resources types:
  - `ServiceAccounts`
  - [RBAC]: `Roles`, `ClusterRoles`, `RoleBindings` and `ClusterRoleBindings`
  - `PodSecurityPolicy` extension

### CODE REVIEW

File: policies.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rigging
+
+import (
+	"context"
+
+	log "github.com/Sirupsen/logrus"
```
Comment (eldios) [inline policies.go]: didn't we decide to use `sirupsen` with a lowercase leading `s` ?

File: roles.go [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rigging
+
+import (
+	"context"
+
+	log "github.com/Sirupsen/logrus"
```
Comment (eldios) [inline roles.go]: same problem with the uppercase `s` here

File: serviceaccount.go [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rigging
+
+import (
+	"context"
+
+	log "github.com/Sirupsen/logrus"
```
Comment (eldios) [inline serviceaccount.go]: uppercase `s` strikes again

File: policies.go (hunk 1 above)
Comment (a-palchikov) [inline policies.go]: I'll do the switch in another PR

**Pull request merged**